config/README.md

```diff
@@ -120,6 +120,7 @@ block_number = 0
 fork_block_number = 0
 chain_id = 1
 # NOTE due to a toml-rs limitation, this value needs to be a string if the desired gas limit exceeds `i64::MAX` (9223372036854775807)
+# `gas_limit = "Max"` is equivalent to `gas_limit = "18446744073709551615"`
```

Comment:
Should we change the default `gas_limit` to be the max also? with u64::MAX that seems ok, since you can still run out of gas so there's not really any added "infinite run duration" risk

Reply:
let's do that separately as it relates to the "revm unlimited gas" conversation